Documentation/sources/content-card-ui/public-classes/UIModels/smallimagetemplate.md

```diff
@@ -0,0 +1,27 @@
+# Data Class - SmallImageTemplate
+
+Class representing a small image template, which implements the [AepUITemplate](aepuitemplate.md) interface.
+
+## Public Properties
+
+| Property   | Type                               | Description                                                  |
+| ---------- | ---------------------------------- | ------------------------------------------------------------ |
+| id         | String                             | The unique identifier for this template.                     |
+| title      | [AepText](./aeptext.md)            | The title text and display settings.                         |
+| body       | [AepText](./aeptext.md)?           | The body text and display settings.                          |
+| image      | [AepImage](./aepimage.md)?         | The details of the image to be displayed.                    |
+| actionUrl  | String?                            | If provided, interacting with this card will result in the opening of the actionUrl. |
```

Comment:
The actionUrl may not always be handled by Messaging. Can we reword this a bit? `Url to be handled when user clicks the card`